common/src/main/java/org/broadleafcommerce/common/resource/service/ResourceMinificationServiceImpl.java

```diff
@@ -107,44 +100,46 @@ public Resource minify(Resource originalResource) {
     @Override
     public Resource minify(Resource originalResource, String filename) {
         if (!getEnabled()) {
-            LOG.trace("Minification is disabled, returning original resource");
+            LOG.trace(MINIFICATION_IS_DISABLED_RETURNING_ORIGINAL_RESOURCE);
             return originalResource;
         }
-        
+
         String type = getFileType(originalResource, filename);
         if (type == null) {
-            LOG.info("Unsupported minification resource: " + filename);
+            LOG.trace("Unsupported minification resource: " + filename);
             return originalResource;
         }
-        
-        byte[] minifiedBytes = null;
-        try (BufferedReader in =
-                new BufferedReader(new InputStreamReader(originalResource.getInputStream(), "utf-8"));
-                ByteArrayOutputStream baos = new ByteArrayOutputStream();
-                BufferedWriter out =
-                        new BufferedWriter(new OutputStreamWriter(baos, "utf-8"));) {
+
+        try (BufferedReader in = new BufferedReader(new InputStreamReader(originalResource.getInputStream(), "utf-8"));
+            ByteArrayOutputStream baos = new ByteArrayOutputStream();
+            BufferedWriter out = new BufferedWriter(new OutputStreamWriter(baos, "utf-8"));) {
+
             minify(in, out, filename, type);
-            
             out.flush();
-            minifiedBytes = baos.toByteArray();
+            return new GeneratedResource(baos.toByteArray(), filename);
         } catch (Exception e) {
-            LOG.warn("Could not minify resources, returned unminified bytes", e);
+            LOG.warn(COULD_NOT_MINIFY_RESOURCES_RETURNED_UNMINIFIED_BYTES, e);
             return originalResource;
         }
-        
-        return new GeneratedResource(minifiedBytes, filename);
     }
     
+    // TODO Change method signature to throw a ResourceMinificationException in 5.4
```

Comment:
I don't think that we should change this to throw an exception ever; we should just straight up handle it  right?

Reply:
We handle it in the call that calls this method by just returning the original file. Technically we could feed the BufferedReader into the BufferedWriter but that seems pretty round about when the calling method can just return the original file.